quantum/rgblight/rgblight.h

```diff
@@ -18,6 +18,7 @@
 
 // DEPRECATED DEFINES - DO NOT USE
 #if defined(RGBLED_NUM)
+#    undef RGBLIGHT_LED_COUNT
```

Comment:
If anything, this should be: 
```suggestion
#if defined(RGBLED_NUM) && !defined(RGBLIGHT_LED_COUNT)
```